rsmtool/notebooks/header.ipynb

```diff
@@ -262,7 +262,7 @@
     "converter_dict = {column: str for column in string_columns}\n",
     "\n",
     "with warnings.catch_warnings():\n",
-    "    warnings.filterwarnings('ignore', category=pd.io.common.DtypeWarning)\n",
```

Comment:
The suggested change below would also apply here. I wonder if you could even import `DtypeWarning` from `rsmtool.reader`.

Reply:
Yep, I did the conditional import here too.